server/s3_delete_object_action.cc

```diff
@@ -76,10 +76,19 @@ void S3DeleteObjectAction::setup_steps() {
   // someone else's object. Whereas deleting metadata first will worst case
   // lead to object leak in motr which can handle separately.
   // To delete stale objects: ref: MINT-602
-  ACTION_TASK_ADD(S3DeleteObjectAction::populate_probable_dead_oid_list, this);
-  ACTION_TASK_ADD(S3DeleteObjectAction::delete_metadata, this);
+
+  /*
```

Comment:
Please match the comment style of existing code, `//`.

Reply:
Updated comment